web-api-db-ctrl/elasticsearch/mapping.json

```diff
@@ -18,7 +18,8 @@
                 "dynamic": false,
                 "properties": {
                     "publication_date": {
-                        "type": "date"
+                        "type": "date",
+                        "ignore_malformed": true
```

Comment:
For consistency, but given how we enforced `publication_date` from the get to go the problem this PR fixes doesn't affect `publication_date` (until we support no longer indexing a property).